app/scripts/HorizontalHeatmapTrack.js

```diff
@@ -239,127 +237,52 @@ class HorizontalHeatmapTrack extends HeatmapTiledPixiTrack {
     sprite.y = this._refYScale(tileY);
   }
 
-  /**
-   * Convert the raw tile data to a rendered array of values which can be represented as a sprite.
-   *
-   * @param tile: The data structure containing all the tile information. Relevant to
-   *              this function are tile.tileData = \{'dense': [...], ...\}
-   *              and tile.graphics
-   */
-  renderTile(tile) {
```

Comment:
This renderTile function now defaults to the parent classes (HeatmapTiledPixiTrack) which removes a lot of code duplication.

Reply:
And, more importantly, lets the horizontal heatmap track use many of the same options as the regular heatmap track.